worldengine/drawing_functions.py

```diff
@@ -530,7 +528,7 @@ def _dynamic_draw_a_mountain(pixels, x, y, w=3, h=3):
         max_leftborder = int(bottomness * w * 1.33)
         if not last_leftborder == None:
             max_leftborder = min(max_leftborder, last_leftborder + 1)
-        leftborder = int(bottomness * w) + random.randint(-2, 2)/2
+        leftborder = int(bottomness * w) + numpy.random.randint(-2, 2)/2
```

Comment:
Yes, it should make use of the RNG used by the ancient map operation. But since it is never called at the moment I decided to be lazy and not adapt it. I will change it now.